### PR TITLE
Lps 201993 | remove $ in language key and update APIEndpointRelevantObjectEntryModelListenerTest test

### PR DESCRIPTION
**Pull request by yanan**
Background
- There is "$" in `post-api-endpoints-retrieve-type-must-be-x` language key
- Update failed integration test

Jira ticket:
- https://liferay.atlassian.net/browse/LPS-201993

### CODE REVIEW

**Comment by yanan** [comment]
ci:test:relevant

**Comment by yanan** [comment]
Failures are not related, and the updated test passed now
image

**Review by magjed4289 — APPROVED** [review]

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -13611,7 +13611,7 @@ position.third=Third
 possible-assignees-must-have-permissions-to-be-assigned-to-the-corresponding-step=Possible assignees must have permissions to be assigned to the corresponding step.
 post=Post
 post-answer=Post Answer
-post-api-endpoints-retrieve-type-must-be-x=POST API endpoints retrieve type must be "${0}."
+post-api-endpoints-retrieve-type-must-be-x=POST API endpoints retrieve type must be "{0}."
```
Comment (jalber786) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:13614]: I think the dot shouldn't be there. It should be `post-api-endpoints-retrieve-type-must-be-x=POST API endpoints retrieve type must be "${0}".`

**Review by jalber786 — COMMENTED** [review]
Just one small comment

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 1 above)
Comment (yanan) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:13614]: Thanks @jalber786 I think so too, but Brian told that the dot should be inside and here [is the link](https://writeshop.com/blog/how-to-use-quotation-marks/) he shared before 
image

**Comment by yanan** [comment]
ci:forward

**Comment by yanan** [comment]
ci:forward:force

**Pull request closed without merging**